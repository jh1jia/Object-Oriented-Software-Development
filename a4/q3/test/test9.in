stdin dropfirst 2 count l doublewords doublewords count 2 count 1 allcaps
Hello world
