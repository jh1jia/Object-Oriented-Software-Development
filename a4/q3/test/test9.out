12O
32O
43O
54O
R3D
R3D
R3D
R3D
